main.py

```
@@ -1,44 +1,73 @@
+        price     = data.get("price", "0")
+        timeframe = data.get("timeframe", "H4")
+        atr       = data.get("atr", "0.00100")
+        if signal == "BUY":
+            message = format_buy_message(price, timeframe, atr)
+        elif signal == "SELL":
+            message = format_sell_message(price, timeframe, atr)
+        else:
+            message = "Pas de signal clair pour le moment"
+        asyncio.run(bot.send_message(chat_id=TELEGRAM_CHAT_ID, text=message))
+        return jsonify({"status": "ok"}), 200
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)